front50-api/src/main/java/com/netflix/spinnaker/front50/api/model/pipeline/Pipeline.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.front50.api.model.pipeline;
+
+import com.netflix.spinnaker.front50.api.model.Timestamped;
+
+import java.util.*;
+
+public class Pipeline implements Timestamped {
```

Comment:
Might be able to get away with Lombok in here? Pretty sure it's been used in other `-api` artifacts.

Reply:
Rather than `@Getter and @Setter`, could probably have just added a `@Data` to the class?